jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskdetails/TaskDetailsPresenter.java

```diff
@@ -53,12 +53,14 @@
 
     private String currentContainerId;
 
+    private TaskSummary currentTaskSummary;
```

Comment:
I'm not sure you need to hold the current task, in terms of testing you could check that the view methods get the expected values.